lua/neo-tree/utils/init.lua

```diff
@@ -881,6 +881,9 @@ M.is_subpath = function(base, path)
   end
   base = M.normalize_path(base)
   path = M.normalize_path(path)
+  if base:match("^.*()/") and path:match("^.*()/") then
+    return base == path
+  end
```

Comment:
This doesn't work, it just turns is_subpath into `base == path`, which is not what the method is supposed to do.